client/main.go

```diff
@@ -24,6 +24,47 @@ var (
 	EventsClient   events.EventsServiceClient
 )
 
+// MetaClient aggregates all the grpc clients available from Tinkerbell Server
+type MetaClient struct {
+	TemplateClient template.TemplateServiceClient
+	WorkflowClient workflow.WorkflowServiceClient
+	HardwareClient hardware.HardwareServiceClient
+	EventsClient   events.EventsServiceClient
+}
+
+// NewMetaClientFromGlobal is a dirty hack that returns a MetaClient using the
+// global variables exposed by the client package. Globals should be avoided
+// and we will deprecated them at some point replacing this function with
+// NewMetaClient. If you are strating a new project please use the last one
```

Comment:
"...are strating..." should probably be starting... ;-)

Reply:
can we skip that in CI? This feels weird to let CI dictate removal of deprecated functions which removes the utility of marking things Deprecated ... I'd expect CI to ignore deprecated functions from within this module :thinking:

Reply:
I was trying to be lazy! I appreciate your attempt to make me a better person!